Source/NSByteCountFormatter.m

```diff
@@ -67,6 +67,13 @@ + (NSString *)stringFromByteCount: (long long)byteCount
   return [formatter stringFromByteCount: byteCount];
 }
 
+- (NSString *)stringForObjectValue: (id)obj
+{
+  long long byteCount = [obj longLongValue];
+  return [[self class] stringFromByteCount: byteCount
```

Comment:
I don't understand this code here, why don't we just call [self stringFromByteCount: byteCount]?

Reply:
You're right.  Done.